robosuite_env.py

```diff
@@ -272,49 +314,57 @@ def __init__(self, rcan_restore_func, rcan_kwargs, env_kwargs):
 
         rcan_kwargs = {} if not rcan_kwargs else rcan_kwargs
         self.rcan = rcan_restore_func(**rcan_kwargs)
-        
+
         env_kwargs = {} if not env_kwargs else env_kwargs
 
         try:
             super(self).__init__(**env_kwargs)
         except TypeError:
             super(RCANRealSawyerLift, self).__init__(**env_kwargs)
-       
+
     def _process_rcan(self, img):
-        print('img shape', img.shape)
         
-        #img = self.rcan.forward(torch.Tensor(img).cuda()).contiguous().detach().cpu().numpy()
-        print('rcan img shape', img.shape)
+        img = self.rcan.forward(torch.Tensor(img).cuda()).contiguous().detach().cpu().numpy()
         img = np.transpose(img[0, :3, ...], (1,2,0))
-        print('1', img.shape)
         img = cv2.resize(img, (84, 84))
-        print('2', img.shape)
         img = np.transpose(img, (2, 0, 1))
-        print (img.shape)
         return img  # 1x3x84x84
 
     def step(self, action):
         try:
             obs, rew, done, info = super(self).step(action)
         except TypeError:
             obs, rew, done, info = super(RCANRealSawyerLift, self).step(action)
-        
+
         img = obs['image']
-        img = self._process_rcan(img) 
+        img = (self._process_rcan(img) + 1.) / 2.
 
-        return img, rew, done, info
+        if 'proprio' in obs:
+            di = {'pixel': {'camera0': img}, 'low_dim': {'robot_state': obs['proprio']}}
+        else:
+            di = {'pixel': {'camera0': img}}
+        
+        return di, rew, done, info
 
     def reset(self):
         d = super(RCANRealSawyerLift, self).reset()
 
         img = d['image']
         img = self._process_rcan(img)
 
-        return img
+        if 'proprio' in d:
+            di = {'pixel': {'camera0': img}, 'low_dim': {'robot_state': d['proprio']}}
+        else:
+            di = {'pixel': {'camera0': img}}
+        
+        return di
+
 
-    
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
+    parser.add_argument('--proprio', action='store_true')
```

Comment:
can you add a help statement that tell which mode requires these args? It helps me be able to run my policies with minimal confusion